.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+    # Remove duplicates
+    filenames = list(set(filenames))
+
+    # Process files
+    for filename in filenames:
+        # Ignore files that appear to be full paths because they start
+        # with a '/' or whatever the OS uses to distinguish directories
+        if filename.startswith(os.sep):
+            continue
+
+        # Create a file path
+        filepath = "{}{}{}".format(os.getcwd(), os.sep, filename)
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)